tests/vm_test.go

```diff
@@ -861,10 +865,11 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 					Expect(err).ToNot(HaveOccurred())
 					Expect(newVM.Spec.RunStrategy).ToNot(BeNil())
 					Expect(*newVM.Spec.RunStrategy).To(Equal(v1.RunStrategyManual))
+					By("Ensuring stateChangeRequests list is cleared")
 					Expect(len(newVM.Status.StateChangeRequests)).To(Equal(0))
 				})
 
-				It("should restart", func() {
+				It("[test_id:2035] should restart", func() {
```

Comment:
I am missing the action restart vm , stop, start 
and wait for VMI to be restarted. observe the UUID has changed to be sure it's really a new instance

Reply:
Fixed! thanks!